kubernetes-extension/src/main/java/org/ballerinax/kubernetes/processors/DeploymentAnnotationProcessor.java

```diff
@@ -268,6 +276,49 @@ private ProbeModel parseProbeConfiguration(BLangExpression probeValue) throws Ku
         return null;
     }
 
+    /**
+     * Get Deployment strategy.
+     *
+     * @param keyValue Value of strategy field of deployment annotation.
+     * @return  DeploymentStrategy..
+     */
+    private DeploymentStrategy getStrategy(BLangRecordLiteral.BLangRecordKeyValueField keyValue)
+            throws KubernetesPluginException {
+        DeploymentStrategy strategy = new DeploymentStrategy();
+
+        if ("string".equals(keyValue.getValue().type.toString())) {
```

Comment:
Can't we use `keyValue.getValue().type.getKind() == TypeKind.STRING` instead of string check here?